SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.iOS.Bindings/Extensions.cs

```diff
@@ -18,6 +19,15 @@ private enum Signal
             SIGSEGV = 11
         }
 
+
+        public CrashesInitializationDelegate(IntPtr handle) : base(handle)
+        {
```

Comment:
Can you add comments why we need those constructors so that we don't delete them?